atef/bin/check.py

```diff
@@ -130,6 +131,11 @@ def build_arg_parser(argparser=None):
         help="Acquire data for comparisons in parallel",
     )
 
+    argparser.add_argument(
+        "-r", "--report-path",
+        help="Path to the report save path, if provided"
+    )
```

Comment:
Have we ever talked about having a config file for atef? If we ever get around to something like that, a default reports directory would be a useful configurable (rather than remembering to -r each time)

Reply:
We haven't, all relevant information has been contained within the config file up until now.

A global config file could be helpful, though I'm not sure what other information we'd want to hold.